acceptance/lib/helper.rb

```diff
@@ -61,7 +61,9 @@ def self.get_option_value(value, legal_values, description,
 
     # precedence is environment variable, option file, default value
     value = ((env_var_name && ENV[env_var_name]) || value || default_value)
-    if value
+    if value == ""
+      value = default_value
```

Comment:
I'm wondering if we should eliminate the `elsif` and just require the value to have been converted to a symbol before calling into the `get_option_value` function.  Seems confusing to only conditionally convert to a symbol from within this function.  I haven't looked at the callers of this function, though, so maybe I'm offbase on this.